gatorgrade/generate/generate.py

```diff
@@ -19,3 +19,10 @@ def write_yaml_of_paths_list():  # expected input: A path list
     # Create a new YAML file with PyYaml in the specific path.
     # Write the default set up messages in YAML file.
     # List the file paths in specific format.
+
+
+def generate_config(target_path_list: List[str], relative_run_path: str = "."):
```

Comment:
The name of the second variable here should be reconsidered, I think. @Ochirsaikhan @Yanqiao4396 what do you think? This path represents *where* on the user's system you will start your search for paths. The value it has does not have to be relative, although the default value is.

Reply:
> The name of the second variable here should be reconsidered, I think. @Ochirsaikhan @Yanqiao4396 what do you think? This path represents _where_ on the user's system you will start your search for paths. The value it has does not have to be relative, although the default value is. I think something like `search_root` could work.

I totally agree with reconsidering the second variable name since it can be an _absolute_ and _relative path_. I think `search_root` works, but I also think `search_directory` should work too.

Reply:
I'll change the variable to search_root, thanks!